packages/app/src/app/pages/Dashboard/Content/routes/Settings/TeamSettings/WorkspaceSettings.tsx

```diff
@@ -405,48 +404,40 @@ export const WorkspaceSettings = () => {
                 onInputValueChange={val => setInviteValue(val)}
                 style={{ paddingRight: 80 }}
               />
-              {team?.subscription?.type === SubscriptionType.TeamPro ? (
-                <Menu>
-                  <Menu.Button
-                    css={css({
-                      fontSize: 3,
-                      fontWeight: 'normal',
-                      paddingX: 0,
-                      position: 'absolute',
-                      top: 0,
-                      right: 2,
-                    })}
-                  >
-                    <Text variant="muted">
-                      {permissionMap[newMemberAuthorization]}
-                    </Text>
-                    <Icon name="caret" size={8} marginLeft={1} />
-                  </Menu.Button>
-                  <Menu.List>
-                    {permissionValues.map(authorization => (
-                      <Menu.Item
-                        key={authorization}
-                        onSelect={() =>
-                          setNewMemberAuthorization(authorization)
-                        }
-                        style={{ display: 'flex', alignItems: 'center' }}
-                      >
-                        <Text style={{ width: '100%' }}>
-                          {permissionMap[authorization]}
-                        </Text>
-                        {newMemberAuthorization === authorization && (
-                          <Icon
-                            style={{}}
-                            name="simpleCheck"
-                            size={12}
-                            marginLeft={1}
-                          />
-                        )}
-                      </Menu.Item>
-                    ))}
-                  </Menu.List>
-                </Menu>
-              ) : null}
+
+              <Menu>
+                <Menu.Button
+                  css={css({
+                    fontSize: 3,
+                    fontWeight: 'normal',
+                    paddingX: 0,
+                    position: 'absolute',
+                    top: 0,
+                    right: 2,
+                  })}
+                >
+                  <Text variant="muted">
+                    {PERMISSION_TEXT_MAP[newMemberAuthorization]}
+                  </Text>
+                  <Icon name="caret" size={8} marginLeft={1} />
+                </Menu.Button>
+                <Menu.List>
+                  {rolesThatUserCanInvite.map(authorization => (
+                    <Menu.Item
+                      key={authorization}
+                      onSelect={() => setNewMemberAuthorization(authorization)}
```

Comment:
Why are we sometimes calling it `roles` and sometimes `authorization`, is it possible to align this?